openshift/templates/tekton/pipelines/complete.yaml

```diff
@@ -118,18 +118,6 @@ objects:
             mounts /zap/wrk to store generated configs and results.
 
       tasks:
```

Comment:
Reorder pipeline so that maintenance only gets turned on if all the builds are successful.  This will reduce downtime.